Source/ui_unix/global.h

```diff
@@ -0,0 +1,10 @@
+#ifndef GLOBAL_H
+#define GLOBAL_H
+
+#include "PS2VM.h"
+#include <QTimer>
+
+CPS2VM* g_virtualMachine = nullptr;
+QTimer *fpstimer = nullptr;
```

Comment:
Do we need to declare the timer as a global variable here? Would probably be nicer to have it as a member of `MainWindow` to reduce its scope.